crypto/multisig.go

```diff
@@ -263,6 +269,11 @@ func MultisigVerify(msg Hashable, addr Digest, sig MultisigSig) (verified bool,
 	var verifiedCount int
 	for _, subsigi := range sig.Subsigs {
 		if (subsigi.Sig != Signature{}) {
+			if batchVerifier != nil {
```

Comment:
maybe move this if statement outside the `for..range` loop, so that it would be executed exactly once ?